crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -189,9 +189,16 @@ pub enum BinnedRenderPhaseBatchSets<BK> {
     MultidrawIndirect(Vec<BinnedRenderPhaseBatchSet<BK>>),
 }
 
+/// A group of entities that will be batched together into a single multi-draw
+/// call.
 pub struct BinnedRenderPhaseBatchSet<BK> {
-    pub(crate) batches: Vec<BinnedRenderPhaseBatch>,
+    /// The first batch in this batch set.
+    pub(crate) first_batch: BinnedRenderPhaseBatch,
```

Comment:
nit: `representative_batch`? `base_batch`? "first" seems somewhat incidental to how the batch set is built vs what it's used for.

Reply:
Well, it does need to be the first one because offsets are calculated relative to it.